### PR TITLE
feat(sandbox): example for feature toggle service

### LINKED ISSUE #396: example for feature toggle service
**Is your feature request related to a problem? Please describe.**
It is observed that we do not have any example for feature toggle service.

**Describe the solution you'd like**
Create an example that explains how to consume the feature toggle service.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yeshamavani**
MIGRATION CHANGE:
migration-20211019091305- example for feature toggle service
migration-20210913104858- example for feature toggle service

gh-396

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #396

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: sandbox/feature-toggle-example/src/application.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      path: '/explorer',
+    });
+    this.component(RestExplorerComponent);
+    this.component(FeatureToggleServiceComponent);
+    unleash.initialize({
+      url: 'http://localhost:4242/api/',
+      appName: 'my-node-name',
+      environment: process.env.APP_ENV,
+      customHeaders: {
+        Authorization:
```
Comment (akshatdubeysf) [inline sandbox/feature-toggle-example/src/application.ts:44]: This should be from environment

File: sandbox/feature-toggle-example/src/application.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    this.static('/', path.join(__dirname, '../public'));
+
+    // Customize @loopback/rest-explorer configuration here
+    this.configure(RestExplorerBindings.COMPONENT).to({
+      path: '/explorer',
+    });
+    this.component(RestExplorerComponent);
+    this.component(FeatureToggleServiceComponent);
+    unleash.initialize({
+      url: 'http://localhost:4242/api/',
```
Comment (akshatdubeysf) [inline sandbox/feature-toggle-example/src/application.ts:40]: Pick from environment

File: sandbox/feature-toggle-example/src/application.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    // Set up default home page
+    this.static('/', path.join(__dirname, '../public'));
+
+    // Customize @loopback/rest-explorer configuration here
+    this.configure(RestExplorerBindings.COMPONENT).to({
+      path: '/explorer',
+    });
+    this.component(RestExplorerComponent);
+    this.component(FeatureToggleServiceComponent);
+    unleash.initialize({
```
Comment (akshatdubeysf) [inline sandbox/feature-toggle-example/src/application.ts:39]: Why not take just the config from user and create unleash instance in the component?

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: sandbox/feature-toggle-example/src/application.ts (hunk 1 above)
Comment (yeshamavani) [reply sandbox/feature-toggle-example/src/application.ts]: done

File: sandbox/feature-toggle-example/src/application.ts (hunk 2 above)
Comment (yeshamavani) [reply sandbox/feature-toggle-example/src/application.ts]: done

**Review by akshatdubeysf — COMMENTED** [review]
Why package-locks changed for inmail and video-conf services?

**Comment by akshatdubeysf** [comment]
@yeshamavani fix sonar and audit

**Comment by yeshamavani** [comment]
those were may be rebase changes @akshatdubeysf

File: sandbox/feature-toggle-example/src/application.ts (hunk 3 above)
Comment (yeshamavani) [reply sandbox/feature-toggle-example/src/application.ts:39]: not needed

**Pull request merged**